### DISCUSSION TITLE
Need help with docker memory usage accumulated over time

### DISCUSSION THREAD

**Discussion by chhayse7en** [anasty17/mirror-leech-telegram-bot#1198] [Q&A]
<img width="740" alt="Screenshot 2022-11-07 115904" src="https://user-images.githubusercontent.com/69116277/200230233-2f4a3ad2-8ed5-4db9-9a17-67cb7ae98c3a.png">
This is the memory usage of the bot with just a few active tasks and the bot uptime is around 12 Days. The memory usage seems to accumulated over time and only reduce when restarting the host. Any suggestions to combat this is appreciated. P.S disk cache of qbittorrent and aria2c is set to 32.

**Comment by anasty17** [comment]
Read more about ram!

**Reply by chhayse7en** [reply]
possible memory leak? the bot memory usage exceeded 2gb of my ram which caused my VPS to start using swap and slow down my VPS in the process. It doesn't make sense for the bot to use this much ram with just a few active tasks.

**Reply by anasty17** [reply]
Report to qbittorrent or aria2c
